tests/e2e/specs/renders-player-assessment.js

```diff
@@ -18,7 +18,7 @@ describe("Player for Assessment quizzes", () => {
       cy.intercept("PATCH", "/session_answers/*", {});
       cy.intercept("PATCH", "/sessions/*", {});
 
-      cy.visit("/quiz/abcd?userId=1");
+      cy.visit("/quiz/abcd?userId=1&apiKey=" + Cypress.env("af-api-key"));
```

Comment:
you don't need to give a valid api key - the check is happening on the backend right? you just need to give some value for the tests.